data/apps.csv

```diff
@@ -1475,3 +1475,4 @@ prompt,geometry,,https://github.com/geometry-zsh/geometry,"A minimalistic, fully
 prompt,Polus,,https://github.com/rashed145/polus-bash-theme,"A fast, lightweight and modern Bash prompt theme written in 100 lines of Bash code."
 prompt,Oh My Posh,https://ohmyposh.dev,https://github.com/jandedobbeleer/oh-my-posh,"From their README: ""The most customisable and low-latency cross platform/shell prompt renderer"" "
 conversion,scss-to-css,https://github.com/adamlui/js-utils/tree/main/scss-to-css#readme,https://github.com/adamlui/js-utils,"Recursively compile all SCSS files into minified CSS."
+conversion,minify.js,https://js-utils.com/minify.js,https://github.com/adamlui/js-utils,"Recursively minify all JavaScript files."
```

Comment:
```suggestion
programming,minify.js,https://js-utils.com/minify.js,https://github.com/adamlui/js-utils,"Recursively minify all JavaScript files."
```
This tool doesn't really convert a file to another format, it's rather a tool for programmers to remove unneeded character in JavaScript files, so IMO it fits better in the "programming" category.

Reply:
I asked ChatGPT if .min.js was different format than .js is how I ended up there, but programming indeed seems a better fit